src/pivot-table/hooks/use-column-width/constants.ts

```diff
@@ -1,12 +1,11 @@
-import { CELL_PADDING, DOUBLE_CELL_PADDING } from "../../components/shared-styles";
+import { DOUBLE_CELL_PADDING } from "../../components/shared-styles";
 import { GRID_BORDER, HEADER_ICON_SIZE, PLUS_MINUS_ICON_SIZE } from "../../constants";
 
-// Size of the icon + left/right padding on icon + gap between icon and text
-export const EXPAND_ICON_SIZE = PLUS_MINUS_ICON_SIZE + DOUBLE_CELL_PADDING + CELL_PADDING;
-// CELL_PADDING as grid gap between header text and menu icon
-export const MENU_ICON_SIZE = CELL_PADDING + HEADER_ICON_SIZE;
-// CELL_PADDING as space between lock icon and header text
-export const LOCK_ICON_SIZE = CELL_PADDING + HEADER_ICON_SIZE;
-
-export const TOTAL_CELL_PADDING = DOUBLE_CELL_PADDING * 2 + GRID_BORDER;
+// Size of the icon + left/right padding on icon
+export const EXPAND_ICON_SIZE = PLUS_MINUS_ICON_SIZE + DOUBLE_CELL_PADDING;
+// DOUBLE_CELL_PADDING as grid gap between header text and menu/lock icon
+export const MENU_ICON_SIZE = HEADER_ICON_SIZE + DOUBLE_CELL_PADDING;
+export const LOCK_ICON_SIZE = MENU_ICON_SIZE;
+// Left and right padding + potential border + 1 px extra buffer
+export const TOTAL_CELL_PADDING = DOUBLE_CELL_PADDING * 2 + GRID_BORDER * 2;
```

Comment:
Could add a new constant here so it is super clear that you don't have 2 borders, but rather one border and one `WIDTH_BUFFER` or something

Reply:
Yes I think that is even better